crates/core/tedge_api/src/group.rs

```diff
@@ -236,7 +236,7 @@ mod tests {
         mock.expect_visit_start_group().never();
         mock.expect_visit_end_group().never();
 
-        let _ = group.accept(&mut mock)?;
+        group.accept(&mut mock)?;
```

Comment:
I'm just surprised that these dummy assignments aren't needed anymore. There must have been some earlier version of clippy where this was required.

Reply:
`let _ = ...` can be used to drop the value when there are no other bindings for it, and it can be used to ignore return values that have `#[must_use]` attribute, like results, so it can be used to ignore errors. There is no lint for it, but for me it looks kind of code-smelly, so I think I'll remove all the `let _ = ` occurrences and see if anything breaks.